OpenKeychain/src/main/java/org/sufficientlysecure/keychain/util/Preferences.java

```diff
@@ -424,6 +424,12 @@ public CloudSearchPrefs createFromParcel(Parcel in) {
         };
     }
 
+    // sync preferences
+
+    public boolean getWifiOnlySync() {
+        return mSharedPreferences.getBoolean(Pref.ENABLE_WIFI_SYNC_ONLY, false);
```

Comment:
Alright, we can have it enabled by default.
I will simply add a `android:defaultValue="true"` in the `sync_preferences` XML file.